cmd/root.go

```diff
@@ -270,7 +270,7 @@ func startWorker(userInput string, failOnSeverity *vulnerability.Severity) <-cha
 				log.Errorf(err.Error())
 			}
 			if isAvailable {
-				log.Infof("New version of %s is available: %s", internal.ApplicationName, newVersion)
+				log.Infof("new version of %s is available: %s (current version is %s)", internal.ApplicationName, newVersion, version.FromBuild().Version)
```

Comment:
nit: I think "current version" is slightly ambiguous here. We've used "current version" in conversation before to refer to the _latest_ released version. I'd suggest something like `"<version> currently installed"`

Reply:
👍🏼  I changed it to `new version of %s is available: %s (currently running: %s)"` so the two versions are not next to each other helping visually.